src/main/java/org/digitalstack/chess/Pawn.java

```diff
@@ -44,7 +46,13 @@ private void setPieceColor(PieceColor value) {
     }
 
     public void move(MovementType movementType, int newX, int newY) {
-        throw new UnsupportedOperationException("Need to implement Pawn.move()") ;
+        if (MOVE.equals(movementType)) {
```

Comment:
if ((MOVE.equals(movementType)) 
    && (this.xCoordinate == newX) 
    && (Math.abs(this.yCoordinate - newY) >= 1))
gives you the same result